crates/sui-bridge-indexer/src/migrations/00000000000000_diesel_initial_setup/up.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE token_transfer_data
+(
+    chain_id                    INT          NOT NULL,
+    nonce                       BIGINT       NOT NULL,
+    sender_address              bytea        NOT NULL,
+    destination_chain           INT          NOT NULL,
+    recipient_address           bytea        NOT NULL,
+    token_id                    INT          NOT NULL,
+    amount                      BIGINT       NOT NULL,
+    PRIMARY KEY(chain_id, nonce)
+);
+CREATE INDEX token_transfer_data_destination_chain ON token_transfer_data (destination_chain);
+CREATE INDEX token_transfer_data_token_id ON token_transfer_data (token_id);
+
+CREATE TABLE token_transfer
+(
+    chain_id                    INT          NOT NULL,
+    nonce                       BIGINT       NOT NULL,
+    status                      TEXT         NOT NULL,
+    block_height                BIGINT       NOT NULL,
+    timestamp_ms                BIGINT       NOT NULL,
+    txn_hash                    bytea        NOT NULL,
+    gas_usage                   BIGINT       NOT NULL,
+    data_source                 TEXT         NOT NULL,
+    PRIMARY KEY(chain_id, nonce, status)
```

Comment:
why is `status` part of the key? that seems excessive to me, thoughts?
I would expect the table to be used with a key that represents everything that has happened to a given bridged asset, as such all `statuses` should be returned no?

Reply:
actually reading some more about composite key I see that we cannot use the `chain_id, nonce` only (I guess we have multiple entries for sui?).
However I do wonder what is the value of this primary key and thinking that some proper indexes may be better.
Unless I do not understand how a primary key is used or can be used here